OpenRA.Renderer.Sdl2/Sdl2GraphicsDevice.cs

```diff
@@ -308,6 +308,34 @@ public void SetLineWidth(float width)
 			ErrorHandler.CheckGlError();
 		}
 
+		public Bitmap TakeScreenshot(Rectangle? area)
+		{
+			var rect = new Rectangle(Point.Empty, size);
+			if (area.HasValue)
```

Comment:
Will it be possible to use this function for https://github.com/OpenRA/OpenRA/issues/2762?